.circleci/config.yml

```diff
@@ -125,16 +119,15 @@ jobs:
     environment:
       CGO_ENABLED: "0"
     docker:
-      - image: circleci/golang:1.9 
+      - image: circleci/golang:1.9
     steps:
       - checkout
       - <<: *exports
-      - <<: *install_kubecfg
       - <<: *install_gcloud_sdk
```

Comment:
Is the gcloud sdk required here?

Reply:
I believe it's used for setting up the GKE cluster

Reply:
it's needed to set up the GKE cluster but that is not being done in this job. This was here because the tests in `pkg/gke/` needed it. Since that is now removed we should remove this line as well.

Reply:
thanks @andresmgot, will do